mongodbatlas/resource_mongodbatlas_backup_compliance_policy.go

```diff
@@ -39,6 +39,16 @@ func resourceMongoDBAtlasBackupCompliancePolicy() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"authorized_user_first_name": {
```

Comment:
Should we update the documentation as well?

Reply:
Good catch Andrea, thank you!